src/linter-jscs.js

```diff
@@ -90,19 +91,23 @@ export default class LinterJSCS {
     }));
 
     this.subscriptions.add(atom.workspace.observeTextEditors((editor) => {
-      editor.getBuffer().onWillSave(() => {
+      const currentEditor = new CompositeDisposable;
+      this.textEditors.set(editor.id, currentEditor);
+      currentEditor.add(editor.getBuffer().onWillSave(() => {
         if (grammarScopes.indexOf(editor.getGrammar().scopeName) !== -1 || this.testFixOnSave) {
           // Exclude `excludeFiles` for fix on save
           const config = this.getConfig(editor.getPath());
-          const exclude = globule.isMatch(
-            config && config.excludeFiles, this.getFilePath(editor.getPath())
-          );
+          const exclude = globule.isMatch(config && config.excludeFiles, this.getFilePath(editor.getPath()));
```

Comment:
I'm not sure how this is passing the CI lint, but this line is past the maximum allowed length with this change.